ios/RNCPagerView.m

```diff
@@ -43,22 +43,22 @@ - (instancetype)initWithEventDispatcher:(id<RCTEventDispatcherProtocol>)eventDis
 - (void)didMoveToWindow {
     // Disable scroll view pan gesture for navigation controller screen edge go back gesture
     if (self.reactViewController.navigationController != nil && self.reactViewController.navigationController.interactivePopGestureRecognizer != nil) {
-           [self.scrollView.panGestureRecognizer requireGestureRecognizerToFail:self.reactViewController.navigationController.interactivePopGestureRecognizer];
-       }
+        [self.scrollView.panGestureRecognizer requireGestureRecognizerToFail:self.reactViewController.navigationController.interactivePopGestureRecognizer];
+    }
 }
 
-- (void)layoutSubviews {
-    [super layoutSubviews];
-    [self calculateContentSize];
+- (void)didUpdateReactSubviews {
+    [self updateContentSizeIfNeeded];
 }
 
-- (void)didUpdateReactSubviews {
-    [self calculateContentSize];
+- (void)reactSetFrame:(CGRect)frame {
+    [super reactSetFrame:frame];
+    [self updateContentSizeIfNeeded];
 }
 
 -(void) insertReactSubview:(UIView *)subview atIndex:(NSInteger)atIndex {
     [super insertReactSubview:subview atIndex:atIndex];
-    [_containerView insertSubview:subview atIndex:atIndex];
```

Comment:
`insertSubview` works differently than `addSubview`, hence why it has been replaced?